serverless.yml

```diff
@@ -0,0 +1,42 @@
+service: sqstodynamo
+
+provider:
+    name: aws
+    region: us-east-1
+    runtime: provided
+    iamRoleStatements:
+        - Effect: Allow
+          Action:
+              - sqs:SendMessage
+              - sqs:ReceiveMessage
+              - sqs:DeleteMessage
+              - sqs:GetQueueAttributes
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+          Resource: '<your-sqs-arn>'
```

Comment:
Kind of like with the dotenv for the source code, can we show the user how to load a .env file prior to running serverless commands? Another alternative is that serverless allows you to [load external yaml files](https://www.serverless.com/framework/docs/providers/aws/guide/variables/), so maybe config is just yaml between this and the application and pulled in via symfony/yaml into the PHP code?